nautobot/circuits/navigation.py

```diff
@@ -33,12 +32,6 @@
                                     "circuits.add_circuit",
                                 ],
                             ),
-                            NavMenuImportButton(
```

Comment:
We should have a `.removed` change fragment about the removal of import buttons.

Reply:
Aye sir